apps/website/src/pages/docs/tile.mdx

```diff
@@ -14,14 +14,130 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tile' />
-
 <LiveExample src='Tile.main.tsx'>
   <AllExamples.TileMainExample client:load />
 </LiveExample>
 
 Tiles are containers that display content and actions pertaining to a single item or project. They should be easy to scan for relevant and actionable information. All text should be concise and placed in a way that clearly indicates informational hierarchy.
 
-## Props
+### Composition API
+
+Users can build Tiles with subcomponents that are fully customizable.
+
+We provide:
+
+- `Tile.Wrapper`
+- `Tile.ThumbnailArea`
+- `Tile.ThumbnailPicture`
+- `Tile.QuickAction`
+- `Tile.TypeIndicator`
+- `Tile.BadgeContainer`
+- `Tile.IconButton`
+- `Tile.Name`
+- `Tile.NameIcon`
+- `Tile.NameLabel`
+- `Tile.Action`
+- `Tile.ContentArea`
+- `Tile.Description`
+- `Tile.Metadata`
+- `Tile.MoreOptions`
+- `Tile.Buttons`
+
+<LiveExample src='Tile.composition.tsx'>
+  <AllExamples.TileCompositionExample client:load />
+</LiveExample>
+
+### Props API
+
+Users can also take advantage of properties provided for `Tile`. A list of available properties can be found [at the end](#props) of this documentation.
+
+<LiveExample src='Tile.props.tsx'>
+  <AllExamples.TilePropsExample client:load />
+</LiveExample>
+
+## Variants
+
+### Actionable
+
+Click on an actionable tile triggers an action.
+
+<LiveExample src='Tile.actionable.tsx'>
+  <AllExamples.TileActionableExample client:load />
+</LiveExample>
+
+Tile can also link to another page.
+
+<LiveExample src='Tile.anchorlink.tsx'>
+  <AllExamples.TileAnchorLinkExample client:load style={{ padding: 50 }} />
+</LiveExample>
+
+A tile might have one or two buttons to perform actions. There cannot be more than two button actions.
+
+<LiveExample src='Tile.quickactions.tsx'>
+  <AllExamples.TileQuickActionsExample client:load />
+</LiveExample>
+
+### Avatar
+
+Avatar tile may look a little different from the rest of the tiles as the thumbnail area instead displays an avatar icon and online status indicator.
+
+<LiveExample src='Tile.avatar.tsx'>
+  <AllExamples.TileAvatarExample client:load />
+</LiveExample>
+
+### Condensed
+
+A condensed tile may include as little as the title, thumbnail, and more menu.
```

Comment:
Since it is called Tile.MoreOptions, I felt "more options" menu would be more appropriate

```suggestion
A condensed tile may include as little as the title, thumbnail, and the more option menu.
```

Reply:
Updated